docs/getting-started-helm.md

```diff
@@ -9,11 +9,19 @@ This section guides you through every step of installing and running Apache Puls
 
 For deploying a Pulsar cluster for production usage, read the documentation on [how to configure and install a Pulsar Helm chart](helm-deploy.md).
 
-## Prerequisite
+:::warning Security Notice
 
-- Kubernetes server 1.21.0+
-- kubectl 1.21.0+
-- Helm 3.10+
+This quickstart guide uses default configurations suitable for development and testing only. The default Helm chart configuration **does not meet production security requirements**. For production deployments, you must review and customize security settings including authentication, authorization, TLS encryption, and network policies.
+
+:::
+
+## Prerequisites
+
+- Kubernetes server 1.21.0+ (1.24+ recommended)
```

Comment:
```suggestion
- Kubernetes server 1.25.0+ (1.32.8 recommended)
```

Reply:
EOL for 1.32.x is Feb 28th, 2026: https://kubernetes.io/releases/patch-releases/#1-32